app/routes/labs._index.mdx

```diff
@@ -0,0 +1,293 @@
+        <div className="usa-paragraph text-base-lightest magin-top-0" style={{
+            fontSize: '1.1rem',
+            zIndex: 1,
+            whiteSpace: 'normal',
+            width: '90%',
+            marginTop: '0px'
+
+          }}>
+
+          The emerging field of multimessenger astronomy combines the observation of disparate messenger signals to reveal unique
+          information about astrophysical sources. The four extrasolar messengers that make up multimessenger astronomy are electromagnetic
+          radiation, gravitational waves, neutrinos, and cosmic rays and the primary multimessenger sources outside the heliosphere are
+          expected to be compact binary pairs (black holes and neutron stars), supernovae, irregular neutron stars, gamma-ray bursts,
+          active galactic nuclei, and relativistic jets. The present study of the these multimessenger sources has become one of the most
+          promising discovery areas of modern astronomy. Identified as one of the central themes of the 2020 Decadal Review, the full
+          realization of time-domain and multi-messenger astronomy now stands to open an entirely new window on the Universe.
+
+          The first joint detection of two merging neutron stars in gravitational waves and electromagnetic radiation on August 17, 2017
+          was a watershed moment for the field of multimessenger astronomy. The information gained from the detection of both messengers
+          enable science that would have been impossible to perform with either signal alone. The discovery demonstrated that multimessenger
+          astrophysics offers a powerful methodology of growing importance, as astronomers combine electromagnetic radiation, gravitational
+          wave radiation, and particle astrophysics observations of cosmic events.
+
+
+          Multimessenger.nasa.gov aims to support this emerging field by consolidating important information on resources available to
+          scientists around the world to enable future discoveries.
+
+        </div>
+
+        <img
+          src={mma}
+          alt="Descriptive Alt Text"
+          style={{
+            width: 'auto',
+            height: '600px',
+            marginLeft: '0px'
+          }}
+        />
+      </div>
+
+      <div className="section-head" style={{marginTop:'150px'}}></div>
+
+      <CardGroup>
+
+        <Card
+          headerFirst
+          gridLayout={{ tablet: { col: 5 } }}
+          containerProps={{ className: '', style: { borderRadius: '10px', borderColor: '#161821' } }}
+          >
+          <CardBody className="padding-4 text-base-lightest" style={{ backgroundColor:'#161821'}}>
+            <h1 className="text-left" style={{paddingTop: '20px'}}>
+              The Decadal Survey Findings on Multi-messenger Astronomy
+            </h1>
+          </CardBody>
+        </Card>
+
+        <Card
+          headerFirst
+          gridLayout={{ tablet: { col: 7 } }}
```

Comment:
It looks like every card has a different width. Is that intentional?

Reply:
Yes, the block quote source and text cards span 5 and 7 columns respectively, while the program and science news each span 4 columns each.